twitter.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Twitter</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="layout">
+            <aside class="left">
+                <img src="./icons/twitter-app-icon.svg" alt="twitter logo" class="logo">
+                <nav class="nav">
+                    <a href="#main" class="hom bar" tabindex="1">Home</a>
+                    <a href="explore.html" class="exp bar" tabindex="2">Explore</a>
+                    <a href="notifications.html" class="not bar" tabindex="3">Notifications</a>
+                    <a href="messages.html" class="mes bar" tabindex="4">Messages</a>
```

Comment:
Have not seen tableindex before. What does it do?

Reply:
Yes, spot on Timmy. What does that do?

Reply:
elements which aren't normally able to be selected using the tab bar, can be given this so when it gains the ability to be navigated to on the browser using tab.